opentasks/src/main/java/org/dmfs/tasks/EditTaskFragment.java

```diff
@@ -714,7 +714,7 @@ public void saveAndExit()
                 }
             }
 
-            if (!TextUtils.isEmpty(TaskFieldAdapters.TITLE.get(mValues)) || mValues.isUpdate())
+            if ((!TextUtils.isEmpty(TaskFieldAdapters.TITLE.get(mValues)) && !TaskFieldAdapters.TITLE.get(mValues).trim().equals("")) || mValues.isUpdate())
```

Comment:
I think changing to `.trim().isEmpty()` would be more performant; it simply checks if the `length` is 0.